unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ChatController/ChannelsUtils.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DCL.Chat
+{
+    public class ChannelsUtils : IChannelsUtils
+    {
+        private const string FEATURE_FLAG_FOR_USERS_ALLOWED_TO_CREATE_CHANNELS = "users_allowed_to_create_channels";
+        private const string VARIANT_FOR_USERS_ALLOWED_TO_CREATE_CHANNELS = "allowedUsers";
+
+        private BaseVariable<FeatureFlag> featureFlags => DataStore.i.featureFlags.flags;
```

Comment:
can you avoid referencing the singleton? you can use `dataStore` field instead

Reply:
Ups! It shouldn't be there... Fixed in https://github.com/decentraland/unity-renderer/pull/3195/commits/19ae21dbf9fccbaf2d825a20fdd8e9a150f31b9f